src/app/geo-header/geo-header.component.ts

```diff
@@ -28,4 +28,23 @@ export class GeoHeaderComponent {
       .dispatch(changeSearchFromAddress($value));
   }
 
+  geolocate() {
+    this.store
```

Comment:
This method is a bit confusing to me. On one hand you ask the store to manage searching state life cycle, but, is it a shared piece of state? I mean, is this piece of state consumed anywhere else? Or is it just ui state that belongs to this component? Maybe it is not elegible to exist in the store.

On the other hand, this component consumes primitives that change state from the service layer, rather than dispatching actions to the store. Why?